docs/src/lang/apalache-operators.md

```diff
@@ -0,0 +1,185 @@
+# Apalache-specific operators
+
+In addition to the standard TLA+ operators described in the previous section, Apalache defines a number of operators, which do not belong to the core language of TLA+, but which Apalache uses to provide clarity, efficiency, or special functionality. These operators belong to the module `Apalache`, and can be used in any specification by declaring `EXTENDS Apalache`.
+
+<a name="Assigmnent"></a>
+## Assigmnent
+
+**Notation:** `v' := e`
+
+**LaTeX notation:** ![coloneq](./img/coloneq.png)
+
+**Arguments:** Two arguments. The first is a primed variable name, the second is arbitrary.
+
+**Apalache type:** `(a, a) => Bool`, for some type `a`
+
+**Effect:** The expression `v' := e` evaluates to `v' = e`. At the level of Apalache static analysis, such expressions indicate parts of an action, where the value of a state-variable in a successor state is determined. See [here](../idiomatic/001assignments.md) for more details about assignments in Apalache.
+
+**Determinism:** Deterministic.
+
+**Errors:** 
+If the first argument is not a primed variable name, or if the assignment operator is used where assignments are prohibited, Apalache statically reports an error.
+
+**Example in TLA+:**
+
+```tla
+x' := 1                   \* x' = 1 
+x' := (y = z)             \* x' = (y = z) 
+x' := (y' := z)           \* x' = (y' = z) in TLC, assignment error in Apalache 
+x' := 1 \/ x' := 2        \* x' = 1 \/ x' = 2 
+x' := 1 /\ x' := 2        \* FALSE in TLC, assignment error in Apalache
+x' := 1 \/ x' := "a"      \* Type error in Apalache 
+(x' + 1) := 1             \* (x' + 1) = 1 in TLC, assignment error in Apalache 
+IF x' := 1 THEN 1 ELSE 0  \* Assignment error in Apalache
+```
+
+**Example in C:**
```

Comment:
Why do we give examples in C in this section, whereas all other sections have examples in Python?

Reply:
Because in Python `a=b`, doesn't return a value unlike `C`, where the value of that expression is `b`.

Reply:
You can't write `a = (b = c)`